pkg/kn/commands/plugin/plugin.go

```diff
@@ -0,0 +1,33 @@
+// Copyright © 2018 The Knative Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package plugin
+
+import (
+	"github.com/knative/client/pkg/kn/commands"
+	"github.com/spf13/cobra"
+)
+
+func NewPluginCommand(p *commands.KnParams) *cobra.Command {
+	pluginCmd := &cobra.Command{
+		Use:   "plugin",
+		Short: "Plugin command group",
+		Long: `Provides utilities for interacting with kn plugins.
```

Comment:
I wonder if s/interacting with/managing/  would sounds better since we're not actually interacting/talking-to the plugins.

Reply:
Currently only interacting since no management commands are present. We will however be adding `add` and `remove` and server side stuff. So interacting and managing is sensible.